src/ng/http.js

```diff
@@ -383,9 +383,49 @@ function $HttpProvider() {
    * array, on request, but reverse order, on response.
    *
    * {@link ng.$http#interceptors Interceptors detailed info}
-   **/
+   */
   var interceptorFactories = this.interceptors = [];
 
+  /**
+   * @ngdoc property
+   * @name $httpProvider#xsrfWhitelistedOrigins
+   * @description
+   *
+   * Array containing URLs whose origins are trusted to receive the XSRF token. See the
```

Comment:
Array containing "origins" that are trusted?
Or if you explicitly accept URLs (but ignore the path etc) then change the name to `xsrfWhitelistedUrls`.

Reply:
I do want to explicitly accept URLs (but ignore the path etc), but I think `xsrfWhitelistedUrls` will be misleading in that case. It would sound like you can whitelist specific URLs.

How about I change the description to _"...containing origins that are trusted..."_ and keep the name as is (and also mention below that path etc will be ignored)?

Reply:
Changing the description works for me.